mcd-sel-fj/mcd-sel-fj.py

```diff
@@ -53,128 +55,113 @@
 
 try:
 
-	# --------------------------------------- #
-	# Getting the Live Exchange Rate from XE  #
-	# --------------------------------------- #
-	
-	# Getting the correct XE webpage (all elements)
-	XE = browser.get(
-	    "https://www.xe.com/currencyconverter/convert/?Amount=1&From=FJD&To=USD")
-	
-	# Scraping the text from the selected element (CSS selector)
-	# findall() and select() methods return a list, indicate index [0] to extract the first element as a string value
-	# Extracting only the number from the text string and converting it to a float value (decimal number)
-	exchange_rate = float(re.findall(
-	    r"[-+]?(?:\d*\.\d+|\d+)", browser.find_element(By.CSS_SELECTOR, "p.result__BigRate-sc-1bsijpp-1.iGrAod").text)[0])
-	
-	print(exchange_rate)
-	print()
-	
-	
-	# -------------------------------------- #
-	# Parsing the data into Dictionary List  #
-	# -------------------------------------- #
-	
-	browser.get("https://fijieats.com/products/listing?supplierId=164")
-	
-	time.sleep(6)
-	
-	category_id_list = []
-	item_list = []
-	price_list = []
-	category_list = []
-	product_list = []
-	
-	
-	# Get ID of each Category div element (currently 0 through 5)
-	submenus = browser.find_elements(
-	    By.XPATH, ".//*[@id='detail_product_page']/div/div[2]/div[2]/div"
-	)
-	
-	for submenu in submenus:
-	    category_id = submenu.get_attribute("id")
-	    category_id_list.extend(category_id)
-	
-	
-	# Outer For Loop iterates through each Category div
-	for ID in category_id_list:
-	
-	    # Inner For Loops scrape (text from the corresponding elements) for
-	    # menu items and prices into individual lists
```

Comment:
The new webpage is much simpler and better structured, making it easier to scrape.
It is no longer necessary to populate items in separate lists.